CONTRIBUTING.md

```diff
@@ -239,13 +239,7 @@ python setup.py develop
 Then use that virtual environment to run the tests, via
 
 ```sh
-python -m nose -s -v niceman
-```
-
-or similarly,
-
-```sh
-nosetests -s -v niceman
+niceman test
```

Comment:
probably better leave original version, just replace `-m nose` with `-m pytest` and `nosetests` with `pytest`.  `niceman test` doesn't expose all handy options (-s -v --pdb etc) so better to use pytest runners directly